src/css/controls/flags/ads.less

```diff
@@ -71,7 +71,7 @@
 
     // Control bar should always be visible during ad playback on touch devices
     // except when jw-flag-ads-vpaid is set
-    &.jw-flag-ads.jw-flag-touch:not(.jw-flag-ads-vpaid) {
+    &.jw-flag-ads.jw-state-playing.jw-flag-touch:not(.jw-flag-ads-vpaid) {
```

Comment:
@robwalch why are we specifying state-playing ? shouldn't the backdrop always be displayed on ads on touch devices ?

Reply:
@karimJWP this is so that the shorter gradient is only shown during the playing state. In other states where the gradient is present, it extends from the control bar up to the display icons to provide a backdrop for all visible control elements.